templates/vite/package.json

```diff
@@ -9,7 +9,8 @@
     "build": "pnpm run xprebuild && tsc -b && vite build",
     "lint": "eslint .",
     "fuels:dev": "fuels dev",
-    "test": "vitest"
+    "test": "vitest",
+    "test:unit": "cd sway-programs && forc test"
```

Comment:
We should probably run these tests in the CI to ensure they pass. Same for all packages.

Reply:
I [tried](https://github.com/FuelLabs/fuels-ts/pull/3100/commits/a69b34972ceb37a6c423103340c90f7f39b6c3da) adding them to CI but this would require us to install `forc` in CI. One thing we could do is use our internal binaries to run these tests only for CI instead of `forc`. Mind if I tackle this in a separate issue?